src/utilities/useI18n.ts

```diff
@@ -1,10 +1,137 @@
+        connectZoneCommand: 'kuma-cp run -c config.yaml',
+        connectZoneDescription2: 'Next, run the following command to connect the Zone.',
+      },
+      kubernetes: {
+        copySaveTokenTitle: 'Copy and save token',
+        copySaveTokenAlertMessage: 'You won’t see the token again! Make sure to save it somewhere now.',
+        prerequisitesTitle: 'Prerequisites',
+        prerequisitesStep1Label: 'Kubernetes cluster',
+        prerequisitesStep1LabelAddendum: 'with load balancer',
+        prerequisitesStep1Description: 'Konnect is compatible with all distributions of Kubernetes, including Minikube, AKS, EKS, GKE and OpenShift.',
+        prerequisitesStep2Label: 'kubectl or oc access',
+        prerequisitesStep2Description: 'You have kubectl or oc (if working with OpenShift) installed and configured to communicate to your Kubernetes TLS.',
+        prerequisitesStep3LinkTitle: 'Helm 3',
+        prerequisitesStep3Tail: 'is installed.',
+        helmTitle: 'Set-up Helm',
+        helmDescription: 'On your local machine, create a namespace in your Kubernetes cluster and pull down the kong Helm repo.',
+        helmStep1Description: 'Create a namespace',
+        helmStep1Command: 'kubectl create namespace kuma-system',
+        helmStep2Description: 'Add the Kong charts repository',
+        helmStep2Command: 'helm repo add kuma https://kumahq.github.io/charts',
+        helmStep3Description: 'Update Helm',
+        helmStep3Command: 'helm repo update',
+        secretTitle: 'Set-up secret',
+        secretDescription1: 'Create a secret.',
+        secretCommand: `echo "
+apiVersion: v1
+kind: Secret
+metadata:
+  name: cp-token
+  namespace: kuma-system
+type: Opaque
+data:
+  token: {token}
+" | kubectl apply -f -`,
+        connectZoneTitle: 'Connect Zone',
+        connectZoneDescription1: 'Copy and paste the following configuration into the Helm chart values.yaml on your local machine.',
+        connectZoneFileName: 'values.yaml',
+        connectZoneConfig: `controlPlane:
+  mode: zone
+  zone: {zoneName}
+  kdsGlobalAddress: {globalKdsAddress}
+  secrets:
+    - Env: KUMA_MULTIZONE_ZONE_KDS_AUTH_CP_TOKEN_INLINE
+      Secret: cp-token
+      Key: token
+ingress:
+  enabled: {zoneIngressEnabled}
+egress:
+  enabled: {zoneEgressEnabled}`,
+        connectZoneDescription2: 'Next, run the following command to connect the Zone.',
+        connectZoneCommand: 'helm install -n kuma-system -f values.yaml kuma kuma/kuma',
+      },
+    },
+    create: {
+      pageTitle: 'Create & connect Zone',
     },
+    delete: {
+      confirmModalText1: 'Are you sure you want to delete the Zone {zoneName}?',
+      confirmModalText2: 'This action cannot be reversed.',
+      confirmModalProceedText: 'Yes, delete',
+      confirmModalTitle: 'Delete Zone',
+      confirmModalErrorText: 'An unexpected error occurred',
+    },
+  },
+}
+
+function translate(path: string, placeholders: Record<string, string> = {}): string {
```

Comment:
why are we inventing translate function. could we use `core/i18n`, I think we should. And make it better if it doesn't handle the cases you guys need.

Reply:
See https://github.com/kumahq/kuma-gui/pull/869#discussion_r1189484381.